src/components/BorrowerProfile/TrusteeDetails.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+	<section v-if="displayTrusteeSection">
+		<p>
+			Trustees are U.S.-based individuals or organizations that refer potential borrowers to Kiva.
+		</p>
+		<h2>
+			{{ trusteeName }}
+		</h2>
+		<dl>
+			<description-list-item
+				:term="'Kiva borrowers'"
+				:details="numLoansEndorsedPublic"
+			/>
+			<description-list-item
+				:term="'Total loans'"
+				:details="totalLoansValueFormatted"
+			/>
+			<description-list-item
+				:term="'Loans defaulted'"
+				:details="numDefaultedLoans"
+			/>
+			<description-list-item
+				:term="'Repayment rate'"
+				:details="repaymentRateFormatted"
+			/>
+		</dl>
+		<a
+			:href="`/partners/${trusteeId}`"
```

Comment:
The url for trustees is `/trustees/${trusteeId}`
Also, use router-link for this:
```
<router-link
	:to="`/trustees/${trusteeId}`"
```